src/webargs/core.py

```diff
@@ -329,9 +239,9 @@ def parse(
             of argname -> `marshmallow.fields.Field` pairs, or a callable
             which accepts a request and returns a `marshmallow.Schema`.
         :param req: The request object to parse.
-        :param tuple locations: Where on the request to search for values.
-            Can include one or more of ``('json', 'querystring', 'form',
-            'headers', 'cookies', 'files')``.
+        :param str location: Where on the request to load values.
+            Can be one of ``('json', 'querystring', 'form', 'headers', 'cookies',
+            'files')``.
         :param callable validate: Validation function or list of validation functions
```

Comment:
I'm going to take a look at this, but my instinct is to both write it all out here and point there in addition... In terms of the documentation's readability, I think it helps to have the list of available values included right there.

Reply:
I've made a change to do both. I was a little concerned that there might be other locations that need to be kept in sync, but it looks like this is the only place where the contents of `__location_map__` are spelled out.

I think this is an okay solution, so I'm going to mark this resolved. But I don't mind discussing further if you like -- e.g. if you're worried that it will get out of sync again in the future.